test/unit/attributes/initializer.spec.js

```diff
@@ -0,0 +1,88 @@
+const { expect } = require('chai');
+const Initializer = require('../../../src/attributes/initializer');
+
+describe('Initializer', () => {
+  let attributes, instance;
+
+  describe('.natives', () => {
+    context('when default attribute is a raw value', () => {
+      describe('.shouldInitialize', () => {
+        beforeEach(() => attributes = { default: 'xyz' });
+
+        it('returns true', () => {
+          expect(Initializer.natives.shouldInitialize(attributes)).to.be.equal(true);
+        });
+      });
+
+      describe('.getValue', () => {
+        beforeEach(() => attributes = { default: 'xyz' });
```

Comment:
Maybe it would be better to call this variable `attributeDescriptor` since it's the descriptor, not the set of attributes, what do you think?